changelog.json

```diff
@@ -1,5 +1,27 @@
 {
   "releases": {
+    "2.1.2": [
+      "[Added] Syntax highlighting support for 20 more programming languages - #7217. Thanks @KennethSweezy!",
+      "[Added] Kitty shell support for macOS - #5162",
+      "[Added] Atom editor support on Windows for beta and nightly channels - #8010. Thanks @Rexogamer!",
+      "[Added] WSL Support - #7821 #7891. Thanks @ADustyOldMuffin @say25!",
```

Comment:
I think this is just enabled for beta https://github.com/desktop/desktop/pull/7922
```suggestion
```